howso/utilities/utilities.py

```diff
@@ -1148,11 +1173,11 @@ def deep_update(base, updates):
 def matrix_processing( # noqa
     matrix: pd.DataFrame,
     normalize: bool = False,
-    normalize_method: Union[Iterable[Union[str, Callable]], str, Callable] = "relative",
+    normalize_method: Iterable[str | Callable, str, Callable] = "relative",
     ignore_diagonals_normalize: bool = True,
     absolute: bool = False,
     fill_diagonal: bool = False,
-    fill_diagonal_value: Union[float, int] = 1,
+    fill_diagonal_value: float | int = 1,
```

Comment:
This should match the docstring or vice-versa. The signature says it defaults to `True` (1 == `True` in this context), but the docstring says it defaults to `False`. Also, I don't know why the signature has `float | int` whereas the docstring is effectively `bool | int`. Probably best to find and ask the original author of this function.

Reply:
I updated the docstring with Jack's help. Thanks!